web-console/src/views/task-view/tasks-view.tsx

```diff
@@ -181,6 +181,10 @@ export class TasksView extends React.PureComponent<TasksViewProps, TasksViewStat
     this.supervisorQueryManager = new QueryManager({
       processQuery: async (query: string) => {
         const resp = await axios.get('/druid/indexer/v1/supervisor?full');
+        const data = resp.data;
+        data.forEach( async (item: any ) => {
```

Comment:
I though the whole point was not to have a query per row of the table, use only the info from `/druid/indexer/v1/supervisor?full`